render/wlr_renderer.c

```diff
@@ -247,9 +249,10 @@ bool wlr_renderer_init_wl_display(struct wlr_renderer *r,
 	return true;
 }
 
-struct wlr_renderer *wlr_renderer_autocreate(EGLenum platform,
-		void *remote_display) {
-	struct wlr_egl *egl = wlr_egl_create(platform, remote_display, NULL);
+struct wlr_renderer *wlr_renderer_autocreate(struct wlr_backend *backend) {
+	int fd = backend_get_drm_fd(backend);
+	struct wlr_egl *egl = wlr_egl_create(EGL_PLATFORM_GBM_KHR,
+			gbm_create_device(fd), NULL);
```

Comment:
We need to handle `gbm_create_device` failures by checking whether it returns NULL here.

Reply:
Done